src/styles/createBreakpoints.js

```diff
@@ -25,7 +25,14 @@ export default function createBreakpoints(breakpoints) {
   }
 
   function down(key) {
-    const value = typeof values[key] === 'number' ? values[key] : key;
+    const endIndex = keys.indexOf(key) + 1;
+
+    if (endIndex === keys.length) {
+      // xl down applies to all sizes
+      return up('xs');
```

Comment:
Humm, maybe we should just warn. What's the point of using?
```js
root: {
  color: 'red',
  [theme.breakpoints.down('xl')]: {
    color: 'blue',
  },
},
```

Isn't it equivalent to?
```js
root: {
  color: 'blue',
},
```

Reply:
The same could be said for `xsUp`, which applies to all sizes (min-width: 0px), which is why I did it this way.

Reply:
Ok, let's not add a warning as someone might want to do it this way to beat specificity. The logic is correct 👍